### PR TITLE
feat(reasoner): supports export to SLS

### PR DESCRIPTION
**Pull request by wangsff**

### CODE REVIEW

File: reasoner/pom.xml [hunk 1]
```diff
@@ -351,6 +351,12 @@
         </exclusions>
       </dependency>
       <!-- hive end -->
+      <dependency>
+        <groupId>com.aliyun.openservices</groupId>
+        <artifactId>log-loghub-producer</artifactId>
+        <version>0.1.4</version>
+      </dependency>
+      <!-- sls end -->
```
Comment (youdonghai) [inline reasoner/pom.xml:359]: missing sls start ?

File: reasoner/runner/runner-common/src/main/java/com/antgroup/openspg/reasoner/sink/KgReasonerSinkType.java [hunk 2]
```diff
@@ -25,5 +25,8 @@ public enum KgReasonerSinkType implements Serializable {
   /** hive table */
   HIVE,
   /** openspg canvas */
-  CANVAS
+  CANVAS,
+
+  /** sls */
+  realTime;
```
Comment (youdonghai) [inline reasoner/runner/runner-common/src/main/java/com/antgroup/openspg/reasoner/sink/KgReasonerSinkType.java:31]: use uppercase

File: reasoner/pom.xml (hunk 1 above)
Comment (wangsff) [reply reasoner/pom.xml:359]: What start?  The above dependency is all I need to run.

File: reasoner/runner/runner-common/src/main/java/com/antgroup/openspg/reasoner/sink/KgReasonerSinkType.java (hunk 2 above)
Comment (wangsff) [reply reasoner/runner/runner-common/src/main/java/com/antgroup/openspg/reasoner/sink/KgReasonerSinkType.java]: fixed

**Review by youdonghai — APPROVED** [review]
LGTM

**Review by baifuyu — APPROVED** [review]

**Pull request merged**